kola/tests/kubernetes/controllerInstall.go

```diff
@@ -0,0 +1,795 @@
+package kubernetes
+
+const controllerInstallScript = `#!/bin/bash
+set -e
+
+# List of etcd servers (http://ip:port), comma separated
+export ETCD_ENDPOINTS=
+
+# Specify the version (vX.Y.Z) of Kubernetes assets to deploy
+export K8S_VER=v1.1.3
```

Comment:
if go `text/template` is used, we can pass it thru the template and maybe test multiple versions of k8s.